src/app/remotes/help-item-editor/help-item-editor.component.ts

```diff
@@ -90,20 +92,20 @@ export class OneCXHelpItemEditorComponent implements ocxRemoteComponent {
         return router.routerState.snapshot.url.split('#')[0]
       })
     )
-    this.applicationId$ = combineLatest([
+    this.productName$ = combineLatest([
       this.appStateService.currentPage$.asObservable(),
       this.appStateService.currentMfe$.asObservable()
     ]).pipe(
       map(([page, mfe]) => {
         if (page?.applicationId) return page.applicationId
-        if (mfe.appId) return mfe.appId
+        if (mfe.productName) return mfe.productName
         return ''
       })
     )
 
```

Comment:
currentPage$.applicationId is no longer usable so I would only use mfe.productName

Reply:
Please remove related tests too